pkgs/by-name/wi/wifitui/package.nix

```
@@ -0,0 +1,24 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub
+}: buildGoModule rec {
+  pname = "wifitui";
+  version = "unstable";
+
+  src = fetchFromGitHub {
+    owner = "greedoftheendless";
+    repo = "wifitui-nix";
+    rev = "main";
```

This is not reproducible, as main is going to receive many updates 😅
In nixpkgs, we are basing things on tag whenever possible (or fixed commits otherwise):
 
```suggestion
    tag = "v${version}";
```